cmds/design.go

```diff
@@ -84,12 +85,12 @@ func (akd *AccountKeysDesign) IsValid([]byte) error {
 	return nil
 }
 
-type GenesisCurrenciesDesign struct {
+type RegisterGenesisCurrencyDesign struct {
```

Comment:
@wyuinche The type "RegisterGenesisCurrencyDesign" actually refers to the CurrencyDesign for the genesis operation. Thus, GenesisCurrencyDesign would be a more appropriate name.

Reply:
@soonkuk I couldn't find what this struct is used for. What name should it be?

Reply:
@wyuinche I suggests "GenesisCurrencyDesign".

Reply:
@soonkuk It has been modified.